cmd/nscgc/main.go

```diff
@@ -20,6 +20,7 @@ var (
 	exhaustiveNeededImages   = flag.String("exhaustive_needed", "", "file that contains image manifests that are needed")
 	exhaustiveNeededImagesV2 = flag.String("exhaustive_needed_v2", "", "file that contains image manifests that are needed, v2")
 	realRunFromDryRun        = flag.String("real_run_from_dry_run", "", "pass dry run log, will delete what dry run marked")
+	useManifestBasedGC       = flag.Bool("use_manifest_based_gc", false, "Whether to cleanup old manifests too")
```

Comment:
we could pass a number of days / hours , like --drop_manifests_older_than=2d
so it's easy to customize and very clear from the invocation line what is supposed to happen.

I think you can use https://pkg.go.dev/time#ParseDuration to parse such a string, maybe flag.Duration exists (not sure)